DataManagementSystem/Utilities/lfc_dfc_db_copy.py

```diff
@@ -508,49 +518,48 @@ def getReplicaData( replicaQueue, name_id_se ):
           replicaCounter += 1
 
           # Get the SSE ID
-          dfc_rep_seId = name_id_se.get( row[CNS_REP_HOST] )
+          dfc_rep_seId = name_id_se.get(row[CNS_REP_HOST])
           # Since the file id were kept consistent, we keep the same
           cns_file_id = row[CNS_REP_FILEID]
 
           # Conversion of the status, just for the file
-          dfc_status = lfc_dfc_status.get( row[CNS_REP_STATUS], 2 )
+          dfc_status = lfc_dfc_status.get(row[CNS_REP_STATUS], 2)
 
           # Modification time
-          dfc_mdate = datetime.utcfromtimestamp( row[CNS_REP_MTIME] ).strftime( '%Y-%m-%d %H:%M:%S' )
+          dfc_mdate = datetime.utcfromtimestamp(row[CNS_REP_MTIME]).strftime('%Y-%m-%d %H:%M:%S')
 
           # Creation time
           # sometimes, row[CNS_REP_CTIME]  is None
           try:
-            dfc_cdate = datetime.utcfromtimestamp( row[CNS_REP_CTIME] ).strftime( '%Y-%m-%d %H:%M:%S' )
-          except:
+            dfc_cdate = datetime.utcfromtimestamp(row[CNS_REP_CTIME]).strftime('%Y-%m-%d %H:%M:%S')
+          except BaseException:
```

Comment:
```suggestion
          except Exception:
```

Reply:
It seems this came back?

Reply:
I don't care about this file, it anyway should not be in this repo at all (honestly I don't know why I touched it).

Reply:
I can remove this file (and others with it), but I will do it in another PR.